tests/integration_tests/test_ray.py

```diff
@@ -574,19 +574,20 @@ def test_ray_audio(tmpdir, dataset_type, ray_cluster_2cpu):
         "window_shift_in_s": 0.02,
         "num_filter_bands": 80,
     }
-    audio_dest_folder = os.path.join(tmpdir, "generated_audio")
-    input_features = [audio_feature(folder=audio_dest_folder, preprocessing=preprocessing_params)]
-    output_features = [
-        binary_feature(),
-    ]
-    run_preprocessing(
-        tmpdir,
-        "dask",
-        input_features,
-        output_features,
-        dataset_type=dataset_type,
-        nan_percent=0.1,
-    )
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
Out of curiosity, can we not use [pytest's tmp_path fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html) here?

Reply:
Let me give this a shot. I recall that there were issues using tmpdir, but I'm seeing some posts online about switching over to tmp_path

Reply:
I wouldn't worry about updating it for now if this is working. The original question was just to satisfy my own curiosity :)